<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="rightWidget" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>161</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QPushButton" name="packButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打包</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secIDLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>02</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unpackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>解包</string>
    </property>
   </widget>
   <widget class="QLabel" name="secIDLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二级ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="modIDLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>模块ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="modIDLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="leftWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>19</y>
     <width>621</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QGroupBox" name="packGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>打包</string>
    </property>
    <widget class="QLabel" name="packDinLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>25</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入裸数据（6字节，空格隔开）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="packDinLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00 01 6E 01 70 00</string>
     </property>
    </widget>
    <widget class="QLabel" name="packDoutLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>491</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输出打包好的数据（10字节）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="packDoutLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="unpackGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>611</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>解包</string>
    </property>
    <widget class="QLabel" name="unpackDoutLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>输出解包好的数据（8字节）</string>
     </property>
    </widget>
    <widget class="QLabel" name="unpackDinLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>输入待解包数据（10字节，空格隔开）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="unpackDoutLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="unpackDinLineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
